packages/server/.env.sample

```diff
@@ -1,16 +1,21 @@
+# Server config
 PORT=8000
+CORS_WHITELIST=https://anj.aragon.org
+
+# DB config
 DB_NAME=court
 DB_USER=
 DB_PASS=
 DB_PORT=5432
 DB_HOST=127.0.0.1
+
+# Admin config
 SECRET_KEY=secret
-CORS_WHITELIST=https://anj.aragon.org
-ADMIN_EMAIL=facu@aragon.one
+ADMIN_EMAIL=you@aragon.one
 ADMIN_PASSWORD=aragon
 
-# Web3 provider
-#PRIVATE_KEY=
```

Comment:
I did this on purpose, to raise awareness that you need this variable, although I was passing it through command line.